### DISCUSSION TITLE
Domain Ideas

### DISCUSSION THREAD

**Discussion by umitcan07** [bounswe/bounswe2024group11#57] [Ideas]
🧵 Let's discuss domain ideas in this thread:

**Comment by umitcan07** [comment]
## Music
- Rap Music (History, Development, Current State)

## Education/Academic
- Universities

## Sports
- Extreme Sports (History, Popular Ones, risks associated

**Comment by umitcan07** [comment]
Board games
 - Chess
 - ...

Games
 (too abroad ?)

**Comment by MucahitErdoganUnlu** [comment]
It is determined as "comics"
